### ISSUE TITLE
Max depth. is written on test failure ("not ok")

### ISSUE THREAD

**Issue by Vanuan** [testem/testem#837]
I'm debugging an issue with "not ok" behavior.
I'm using `testem ci -l chrome -R tap`.
In `testem/lib/runners/browser_test_runner.js` (`onTestResult`). I've encountered this message: 
{"passed":"Max depth.","message":"Max depth.","stack":"Max depth."}

Do you have any insight who writes "Max depth." and why?
With it, I can't see test failure output.

**Comment by bantic** [comment]
@Vanuan This is from the [decycling code](https://github.com/testem/testem/blob/4dd317a7f1dec9a5cfea24c8ca672813462fa090/public/testem/decycle.js#L51) that is used to serialize an object before it is `postMessage`'d to the iframe that Testem manages on the test page.

This is most likely to come from a `console.log` statement in your tests that is logging a deeply-nested/complicated object. Is that possible?

**Comment by Vanuan** [comment]
Well, does it mean that mocha outputs a deeply nested object on failure?

**Comment by Vanuan** [comment]
But in browser, I can see failure messages just fine.

**Comment by bantic** [comment]
@Vanuan Can you shed some more light on what the test looks like and what the failure looks like? If you have a minimal working sample that would help to debug.

**Comment by Vanuan** [comment]
```
describe.only('Something', function() {
  it('fails', function() {
    expect(false).to.eql(true);
  })
})

```

Mocha & expect:
```
    "mocha": "1.8.x",
    "expect.js": "0.2.x",
```

**Comment by bantic** [comment]
@Vanuan Can you replicate the issue using the [simple mocha example](https://github.com/testem/testem/tree/master/examples/mocha_simple)? I force-installed mocha@1.8.2 and expect.js@0.2.0 and it appears to still display the test failure properly:

![image](https://cloud.githubusercontent.com/assets/2023/13958626/2aaaa0fe-f027-11e5-9b69-e2612603f042.png)

Do you get the same results with newer mocha/expect? (mocha@1.8.2 is 3 years old and expect.js@0.2.0 is 3.5 years old).

**Comment by Vanuan** [comment]
Debugging deeper, here's decycle function argument:

```
{
   "type":"emit-message",
   "data":[
      "test-result",
      {
         "passed":0,
         "failed":1,
         "total":1,
         "pending":0,
         "id":1,
         "name":"Something fails ",
         "items":[
            {
               "passed":false,
               "message":"expected false to sort of equal true",
               "stack":"Error: expected false to sort of equal true\n    at Assertion.assert (http://localhost:7357/node_modules/expect.js/expect.js:99:13)\n    at Assertion.eql (http://localhost:7357/node_modules/expect.js/expect.js:214:10)\n    at Context.<anonymous> (http://localhost:7357/spec/unit/test_spec.js:4:22)\n    at Test.Runnable.run (http://localhost:7357/node_modules/mocha/mocha.js:4047:32)\n    at Runner.runTest (http://localhost:7357/node_modules/mocha/mocha.js:4405:10)\n    at http://localhost:7357/node_modules/mocha/mocha.js:4451:12\n    at next (http://localhost:7357/node_modules/mocha/mocha.js:4331:14)\n    at http://localhost:7357/node_modules/mocha/mocha.js:4340:7\n    at next (http://localhost:7357/node_modules/mocha/mocha.js:4288:23)\n    at http://localhost:7357/node_modules/mocha/mocha.js:4308:5"
            }
         ]
      }
   ]
}
```

**Comment by Vanuan** [comment]
maxDepth is 4, so it's clearly can't reach `stack` and `message`.

**Comment by bantic** [comment]
@Vanuan Can you make sure you're on latest testem? Maxdepth changed from 4 to 5 in #813 which was released in [v1.4.2](https://github.com/testem/testem/releases/tag/v1.4.2).

**Comment by Vanuan** [comment]
Yeah, this is it! I was using 1.4.0
